### PR TITLE
added documentation for visibilities

### PR DESCRIPTION
**Pull request by vihangvk**

### CODE REVIEW

File: content/usage/extensions/estafette-extensions.md [hunk 1]
```diff
@@ -144,6 +144,8 @@ The `gke` extension is used to deploy an application to Kubernetes Engine. It ge
...
 If you don't want to use the default sidecar, you should add the `injecthttpproxysidecar: false` field to the deployment, then it will not be injected during the development.
 
+Default visibility is set to private, which creates an internal ingress and it makes the service accessible from within same network. Other supported visibilities are `public-whitelist`, `iap`, `apigee` and `esp`. For visibility `esp`, `useGoogleCloudCredentials` and `disableServiceAccountKeyRotation` should be set to `true`, `espOpenapiYamlPath` must be set and exactly one `host` must be provided. For visibility `iap`, base64 encoded values of `iapOauthClientID` and `iapOauthClientSecret` must be provided. For visibility `apigee`, `authsecret` under `request` must refer to a Secret containing certificate for client authentication.
+
```
Comment (JorritSalverda) [inline content/usage/extensions/estafette-extensions.md:148]: Perhaps you can add a short yaml snippet for each visibility value that requires extra parameters?

**Comment by vihangvk** [comment]
@JorritSalverda Please check. And is there a way to see how it will look in html?

File: content/usage/extensions/estafette-extensions.md (hunk 1 above)
Comment (vihangvk) [reply content/usage/extensions/estafette-extensions.md]: done

**Comment by JorritSalverda** [comment]
Nope, there's no preview site to view what it will look like. I'll just merge it and then we'll see what it looks like.

**Pull request merged**